docs/guides/authjs-migration.mdx

```diff
@@ -69,7 +69,9 @@ This guide shows how to migrate an application using Auth.js (formerly NextAuth.
 
   ### Wrap your Next.js app in `<ClerkProvider />`
 
-  You will need to remove the `<SessionProvider session={session}>` provider from Auth.js and replace it with the `<ClerkProvider />` provider from Clerk. `<ClerkProvider />` will need to wrap your application, or wrap the portion of your app where you want to handle authentication.
+  Remove the `<SessionProvider session={session}>` provider from Auth.js and replace it with `<ClerkProvider />`.
+
+  The [`<ClerkProvider>`](/docs/components/clerk-provider) component provides session and user context to Clerk's hooks and components. It's recommended to wrap your entire app at the entry point with `<ClerkProvider>` to make authentication globally accessible. See the [ClerkProvider docs](/docs/components/clerk-provider) for other configuration options.
```

Comment:
`<ClerkProvider>` should always be formatted as such.  Just "ClerkProvider" is not accurate, and not used anywhere in the docs. Any piece of code in the docs should **always** be formatted correctly.

But I prefer not to put the docs' title in the link

```suggestion
  The [`<ClerkProvider>`](/docs/components/clerk-provider) component provides session and user context to Clerk's hooks and components. It's recommended to wrap your entire app at the entry point with `<ClerkProvider>` to make authentication globally accessible. See the [reference doc](/docs/components/clerk-provider) for other configuration options.
```